src/main/scala/firrtl/annotations/Named.scala

```diff
@@ -14,6 +14,13 @@ sealed trait Named {
   def serialize: String
 }
 
+// TODO Is there a better way to just indicate the top?
+// Perhaps this should be a termporary thing that gets turned into CircuitName below
+final case object CircuitTopName extends Named {
+  def name: String = "CircuitTop"
+  def serialize: String = name
+}
+
 final case class CircuitName(name: String) extends Named {
```

Comment:
I'm thinking that should be done in a separate commit because it touches quite a bit of stuff and the conversion isn't completely mechanical. Things like inline instances work with circuitName[.moduleName[.componentName]] are kind of strange to change since circuitName would now be ignored. I think we should do this separately so we can discuss it independently of all of this